.travis.yml

```diff
@@ -0,0 +1,27 @@
+# This file was generated automatically from conda-smithy. To update this configuration,
+# update the conda-forge.yml and/or the recipe/meta.yaml.
+
+language: generic
+
+
+
+matrix:
+  include:
+    - env: CONFIG=linux_aarch64_ UPLOAD_PACKAGES=True PLATFORM=linux-aarch64 DOCKER_IMAGE=quay.io/condaforge/linux-anvil-aarch64
+      os: linux
+      arch: arm64
+      dist: focal
+
+    - env: CONFIG=linux_ppc64le_ UPLOAD_PACKAGES=True PLATFORM=linux-ppc64le DOCKER_IMAGE=quay.io/condaforge/linux-anvil-ppc64le
```

Comment:
@hmaarrfk @conda-forge/arm-arch when you say to skip the tests for ppc64le, would that just be a case of removing this section of the `.travis.yml` file? Or, are there other things that would need changed?

Reply:
@mattpitkin, I think you want something like this:

```patch
diff --git a/recipe/meta.yaml b/recipe/meta.yaml
index 01bf8a2..d1fc2b0 100644
--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -48,7 +48,8 @@ test:
     - tests/test_data/test_de430.par
     - tests/test_data/test_de430.tim
   commands:
-    - tempo2 -f tests/test_data/test_de430.par tests/test_data/test_de430.tim
+    # [explanation for why ppc64le is skipped, with link to upstream bug report]
+    - tempo2 -f tests/test_data/test_de430.par tests/test_data/test_de430.tim  # [not ppc64le]

 about:
   home: https://bitbucket.org/psrsoft/tempo2/
```

Reply:
yes